sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/FileAPITests.groovy

```diff
@@ -124,18 +124,12 @@ class FileAPITests extends APISpec {
         resp.getValue().getSmbProperties().getFileId()
     }
 
-    @Unroll
     def "Create file with args error"() {
         when:
-        primaryFileClient.createWithResponse(maxSize, null, null, null, metadata, null, null)
+        primaryFileClient.createWithResponse(-1, null, null, null, testMetadata, null, null)
         then:
         def e = thrown(StorageException)
-        FileTestHelper.assertExceptionStatusCodeAndMessage(e, statusCode, errMsg)
-
-        where:
-        maxSize | metadata                                      | statusCode | errMsg
-        -1      | testMetadata                                  | 400        | StorageErrorCode.OUT_OF_RANGE_INPUT
-        1024    | Collections.singletonMap("testMeta", "value") | 403        | StorageErrorCode.AUTHENTICATION_FAILED
```

Comment:
Is this no longer an issue because of the way we signed the metadata?

Reply:
It tests failed cases. Metadata with upper case is no longer invalid. So remove the where condition and only test invalid size.